### PR TITLE
ncm-iptables: clean up documentation

### PR DESCRIPTION
**Pull request by wdpypere**
- some structure work
 - cannot find extra documentation mentioned, so removed mentioning of it
 - some minor spelling fixes.

### CODE REVIEW

File: ncm-iptables/src/main/perl/iptables.pod [hunk 1]
```diff
@@ -143,79 +137,55 @@ system for DDoS attacks it is worthwhile, for machines which are not
...
-
 =head1 FILES
 
-=head2 /etc/sysconfig/iptables:
+=head2 C<< /etc/sysconfig/iptables >>:
 
 I<IPTABLES> firewall configuration file policy.
 
-=head2 pro_declaration_component_iptables.tpl:
+=head2 pro_declaration_component_iptables.pan:
```
Comment (ned21) [inline ncm-iptables/src/main/perl/iptables.pod]: These two files also no longer exist?

File: ncm-iptables/src/main/perl/iptables.pod [hunk 2]
```diff
@@ -10,15 +10,11 @@ iptables: Setup the IPTABLES firewall rules.
...
 
 The I<IPTABLES> component perform the setup of the
-B</etc/sysconfig/iptables> configuration file and restarts the
+C<< /etc/sysconfig/iptables >> configuration file and restarts the
 iptables service.
 
 =head1 SYNOPSIS
 
-Note: a detailed HOWTO for this component, including examples, is
-found in /usr/share/doc/ncm-iptables-2.3.13/ncm-iptables-howto.html and in the FIO twiki pages
```
Comment (ned21) [inline ncm-iptables/src/main/perl/iptables.pod]: Is there a version of the 2.3.13 rpm around anywhere? Perhaps we could check if that contains this file? Perhaps the source was lost in the migration to maven?

File: ncm-iptables/src/main/perl/iptables.pod (hunk 1 above)
Comment (wdpypere) [inline ncm-iptables/src/main/perl/iptables.pod]: indeed, I'll remove them as well.

File: ncm-iptables/src/main/perl/iptables.pod (hunk 2 above)
Comment (wdpypere) [inline ncm-iptables/src/main/perl/iptables.pod]: if we can get the old rpm, I'm happy to read anything that is in it. maybe it also still contains the 2 missing pan/tpl files.

File: ncm-iptables/src/main/perl/iptables.pod [hunk 3]
```diff
@@ -31,25 +27,21 @@ default targets are supported.
...
-with multiple chains E.g. chain "LocalRules" is not the same as
+needs to - N.B. This means that you need to spell target names
+consistently and with identical capitalisation otherwise you will end up
+with multiple chains. E.g. chain "LocalRules" is not the same as
 "localrules".
 
 Duplicated entries in the component resource declaration are
 ignored. For each configured table, the chains are added to the
-B</etc/sysconfig/iptable> in order, the relative order among the rules
+C<< /etc/sysconfig/iptable >> in order, the relative order among the rules
```
Comment (stdweird) [inline ncm-iptables/src/main/perl/iptables.pod]: `s` missing?

File: ncm-iptables/src/main/perl/iptables.pod (hunk 2 above)
Comment (stdweird) [inline ncm-iptables/src/main/perl/iptables.pod]: maybe ask around on the mailing list (we don't have these)

Comment (wdpypere) [inline ncm-iptables/src/main/perl/iptables.pod]: the oldest I can find in our repos is 13.2.0 which does not contain any of the mentioned files.

Comment (wdpypere) [inline ncm-iptables/src/main/perl/iptables.pod]: OK, I'll ask on the mailing list.

Comment (stdweird) [inline ncm-iptables/src/main/perl/iptables.pod]: the declaration file is now schema, but i have no clue what the functions are about.

File: ncm-iptables/src/main/perl/iptables.pod (hunk 3 above)
Comment (wdpypere) [inline ncm-iptables/src/main/perl/iptables.pod]: Yes! thx. I'll fix it.

**Comment by wdpypere** [comment]
@ned21 the old examples were recovered and added again.

**Comment by ned21** [comment]
Thanks, @wdpypere. I don't think it really makes sense to split these two changes across two commits so would you be OK to squash them into a single commit?

**Comment by wdpypere** [comment]
@ned21 squashed the commits.

**Pull request merged**